scripts/tracker.lua

```diff
@@ -55,11 +74,82 @@ function Tracker.newTracker(trackerType, trackerList)
         newTracker.userCanEnable = false
         newTracker.enabled = false
         newTracker.realtimeCamera = true
+    elseif trackerType == "cityblock" then
+        -- cityblock-specific data
+        newTracker.cityBlock = Tracker.cityBlock:new()
+        newTracker.minPos = { x = -5, y = -5 }
+        newTracker.maxPos = { x = 5, y = 5 }
+        Tracker.recalculateCityBlock(newTracker)
     end
 
     return newTracker
 end
 
+---comment  find the city block containing the position and relocate to that one
```

Comment:
```suggestion
---find the city block containing the position and relocate to that one
```

The comment does not require some 'operator' (as far as I am aware). If it does it would have been `@comment` I suppose?

Also applicable on other places